features/compose_message.feature

```diff
@@ -0,0 +1,30 @@
+Feature: As a visitor.
+  So that I can use the service.
+  I would like to be able to go to login page and login on my account.
+
+Background:
+Given the following users exist
+| name  | email           | password     |
+| felix | felix@telia.com | fisk1234     |
+
+Scenario: As I login
+  Given I am on the index page
+  And I click "Login"
+  And I should see "Log in"
+  And I fill in "Email" with "felix@telia.com"
+  And I fill in "Password" with "fisk1234"
+  And I click "Log in"
+  Given I am logged in as "felix"
```

Comment:
This is way to many steps to get in state. This is a perfect example where Warden helpers should be used